src/core/Tooltip/index.tsx

```diff
@@ -35,9 +51,26 @@ const Tooltip = (props: TooltipProps): JSX.Element => {
     props,
   });
 
+  const content = (
+    <>
+      {title}
+      {subtitle && <Subtitle>{subtitle}</Subtitle>}
+    </>
+  );
+
+  const leaveDelay = inverted || sdsStyle === "dark" ? 0 : 500;
+
   // (thuang): {...rest} needs to be first, otherwise it will overwrite latter
   // props
-  return <RawTooltip {...rest} interactive classes={{ arrow, tooltip }} />;
+  return (
+    <RawTooltip
+      {...rest}
```

Comment:
I think `{...rest}` needs to go last in the props, so the callsite can overwrite it properly 🙆‍♂️ Thank you!

Reply:
There's a comment right above this line: `// (thuang): {...rest} needs to be first, otherwise it will overwrite latter props`, should I update that as well?